src/main/scala/io/iohk/ethereum/Mantis.scala

```diff
@@ -2,9 +2,12 @@ package io.iohk.ethereum
 
 import io.iohk.ethereum.nodebuilder.{StdNode, TestNode}
 import io.iohk.ethereum.utils.{Config, Logger}
+import java.util.logging.LogManager
 
 object Mantis extends Logger {
   def main(args: Array[String]): Unit = {
+    LogManager.getLogManager().reset();
```

Comment:
Can you explain why you added this instruction?

Reply:
Yes good question. This disables `java.util.logging` which is used in some older parts of the UPnP library. This logging can't be controlled via logback.xml and isn't very informative anyway.

Reply:
since the reason is obscure and not easily deduced - maybe a smol note?